src/storage/tests.rs

```diff
@@ -107,7 +119,7 @@ mod map_test {
         let mut w = World::new();
         let mut c = create(&mut w);
 
-        c.insert(ent(1 << 25), Comp(7));
+        let _ = c.insert(ent(1 << 25), Comp(7));
```

Comment:
I left the `let _ =` here because the panic we're trying to catch shouldn't come from the failed insert.